<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>636</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>vABS</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Query Result</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>461</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab{
     height: 60px;
     width:100px;
}</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Equality Query</string>
      </attribute>
      <widget class="QPushButton" name="querybtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Patient ID:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Check-Up No:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pidlineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>211</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="aidlineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>72</y>
         <width>211</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Range Query</string>
      </attribute>
      <widget class="QPushButton" name="querybtn_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Patient ID:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pidlineEdit_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>211</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Check-Up No:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>247</x>
         <y>75</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="aidlineEdit_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>72</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="aidlineEdit_3">
       <property name="geometry">
        <rect>
         <x>269</x>
         <y>72</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Check-Up No.</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Diagnose</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start Time</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Time</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Direct infection of joint in infectious and parasitic diseases classified elsewhere</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>1994-07-22 07:12:50.407</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>1994-07-28 12:29:05.827</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>Disseminated intravascular coagulation [defibrination syndrome]</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>2003-03-02 09:59:11.540</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>2003-03-22 01:31:32.883</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>haaaaaaaaaaaaaaaa</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>2007-07-03 20:00:07.223</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>2007-07-08 14:23:38.367</string>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>EHR Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-1</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>25</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome Doctor !&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>53</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>vABS System</string>
     </property>
    </widget>
    <widget class="QPushButton" name="account_btn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: blue;
border: 0px; </string>
     </property>
     <property name="text">
      <string>My Account</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="out_btn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: blue;
border: 0px;
</string>
     </property>
     <property name="text">
      <string>Sign out</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="vbutton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>frame</zorder>
   <zorder>label_8</zorder>
   <zorder>frame_2</zorder>
   <zorder>label_20</zorder>
   <zorder>vbutton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>Information(&amp;I)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menuInformation"/>
   <addaction name="menuHelp"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
